### PR TITLE
 LPD-34678  Return Details Summary - Display Order Shipping Costs

### PR DESCRIPTION
**Pull request by ghost**
https://liferay.atlassian.net/browse/LPD-34678

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/commerce_return/general.jsp [hunk 1]
```diff
@@ -84,6 +84,7 @@ CommerceOrder commerceOrder = commerceReturnEditDisplayContext.getCommerceReturn
 								</p>
 							</c:if>
 						</commerce-ui:info-box>
+
```
Comment (dsitu) [inline modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/commerce_return/general.jsp:87]: Dont think we need a space here.

File: modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/commerce_return/general.jsp [hunk 2]
```diff
@@ -202,6 +203,13 @@ CommerceOrder commerceOrder = commerceReturnEditDisplayContext.getCommerceReturn
 					<div class="col-6 col-md-3">
 						<span class="summary-table-item"><%= commerceReturn.getRequestedItems() %></span>
 					</div>
+					<div class="col-6 col-md-9">
```
Comment (dsitu) [inline modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/commerce_return/general.jsp:206]: add a space here.

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:report:77724603

**Comment by CrescenzoRegaLF** [comment]
Hey @dsitu the failures do not depend on the changes made in the pr, after the review if there are no changes to be made you can forward, thanks

**Review by dsitu — APPROVED** [review]

**Comment by dsitu** [comment]
ci:forward

**Comment by dsitu** [comment]
ci:forward:force

**Pull request closed without merging**